### PR TITLE
smali 2.1.3

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by DomT4** [comment]
👍

File: Formula/smali.rb [hunk 1]
```diff
@@ -27,15 +27,11 @@ def install
...
     libexec.install resource("smali"), resource("baksmali"), "smali-#{version}.jar" => "smali.jar"
 
-    inreplace "#{libexec}/smali" do |s|
-      s.gsub! /^libdir=.*$/, "libdir=\"#{libexec}\""
-    end
-    inreplace "#{libexec}/baksmali" do |s|
-      s.gsub! /^libdir=.*$/, "libdir=\"#{libexec}\""
-    end
+    inreplace libexec/"smali", /^libdir=.*$/, "libdir=\"#{libexec}\""
+    inreplace libexec/"baksmali", /^libdir=.*$/, "libdir=\"#{libexec}\""
```
Comment (ilovezfs) [inline Formula/smali.rb]: @DomT4 I suppose these should be combined since the replacements are actually the same. Another thing to add to audit! :)

Comment (DomT4) [inline Formula/smali.rb]: Meh, I don't mind this too much. The alternative isn't any more condensed really.

Comment (ilovezfs) [inline Formula/smali.rb]: Yeah, it's about not repeating the meaningful part twice, as opposed to space savings,

**Comment by ilovezfs** [comment]
@DomT4 how's that?

**Comment by DomT4** [comment]
Seems alright. Nice work.

**Pull request closed without merging**